lib/hypernova/blank_renderer.rb

```diff
@@ -31,4 +31,8 @@ def key
   def name
     job[:name]
   end
+
+  def id
+    SecureRandom.uuid
```

Comment:
I noticed occasionally this returns different values for the same component. Might want to change this to

```
def id
  @id ||= SecureRandom.uuid
end
```

To prevent this from happening.